api/v1/controllers/lenses.js

```diff
@@ -287,7 +287,10 @@ module.exports = {
           extraAttributes = ['lensLibrary'];
         }
 
-        u.findByKey(helper, req.swagger.params, extraAttributes)
+        const params = req.swagger.params;
+        const opts = u.buildFieldList(params, helper.model);
+
+        u.findByKey(helper, params, extraAttributes)
```

Comment:
seems like overkill to have to change the subject and lens controllers when the issue never happens on those models.